test/spec.yml

```diff
@@ -385,6 +391,20 @@
         err: Invalid API key
         exit: 1
 
+- test: Test login empty message (bad-path)
+  steps:
+    -   in: bn create node8 stormysummer
```

Comment:
Why does this create a project when good-path doesn't?

Reply:
My mistake, I was using another test as a framework and missed this.